src/Service/Services/RequestValidator.cs

```diff
@@ -177,13 +178,14 @@ public static void ValidateStoredProcedureRequest(string? primaryKeyRoute)
         }
 
         /// <summary>
-        /// Validates the primarykeyroute is populated with respect to an Update or Upsert operation.
+        /// Validates the request body is a valid JSON string and is not a JSON array since we only
+        /// support point operations.
         /// </summary>
         /// <param name="requestBody">Request body content</param>
         /// <exception cref="DataApiBuilderException">Thrown when request body is invalid JSON
         /// or JSON is a batch request (mutation of more than one entity in a single request).</exception>
         /// <returns>JsonElement representing the body of the request.</returns>
-        public static JsonElement ParseRequestBody(string requestBody)
+        public static JsonElement TryValidateAndParseRequestBody(string requestBody)
```

Comment:
this method does not use "Try" semantics. 
```suggestion
        public static JsonElement ValidateAndParseRequestBody(string requestBody)
```